StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -371,7 +371,8 @@ extension PaymentSheetFormFactory {
             isSelectedByDefault: false,
             didToggle: didToggle
         )
-        return PaymentMethodElementWrapper(element) { _, params in
+        return PaymentMethodElementWrapper(element) { checkbox, params in
```

Comment:
Nit: There can be two checkboxes, or at least it seems unclear what checkbox this refers to, can we make it something more specific to what checkbox it is?